src/python/pants/backend/project_info/cloc.py

```diff
@@ -115,6 +116,7 @@ async def run_cloc(
         description=(
             f"Count lines of code for {pluralize(len(sources_snapshot.snapshot.files), 'file')}"
         ),
+        level=LogLevel.DEBUG,
```

Comment:
Little reason to log this because directly underneath we will put the results of running the process.